tools/image-info

```diff
@@ -1670,11 +1943,39 @@ def read_authselect_conf(tree):
     return result
 
 
+def read_resolv_conf(tree):
+    """
+    Read /etc/resolv.conf.
+
+    Returns: a list of uncommented lines from the /etc/resolv.conf.
+
+    An example return value:
+    [
+        "search redhat.com",
+        "nameserver 192.168.1.1",
+        "nameserver 192.168.1.2"
+    ]
+    """
+    result = []
+
+    with contextlib.suppress(FileNotFoundError):
+        with open(f"{tree}/resolv.conf") as f:
+            for line in f:
+                line = line.strip()
+                if not line:
+                    continue
+                if line[0] == "#":
+                    continue
+                result.append(line)
+
+    return result
+
+
 def append_filesystem(report, tree, *, is_ostree=False):
     if os.path.exists(f"{tree}/etc/os-release"):
         report["packages"] = rpm_packages(tree, is_ostree)
         if not is_ostree:
-            report["rpm-verify"] = rpm_verify(tree)
+            report["rpm-verify"] = rpm_verify(tree, is_ostree)
```

Comment:
This is a little confusing.  Is `rpm_verify()` ever called with `is_ostree=True`?

Reply:
You are right, the `rpm_verify()` is called only from this place and it is confusing. I'm not sure why it is not called for ostree images. The part that I added, related to not installed docs does not call rpm in the chroot and takes the `--dbpath` as an argument. I could do a few things:

1. move the new functionality I added related to not installed docs out of `rpm_verify()`.
2. try running `rpm_verify()` also on ostree and not use chroot to run it.

Reply:
I imagine `rpm --verify` doesn't make sense on ostree since it's all immutable.

Quoting the commit message that added support for ostree to `image-info` (0479bfc78):
> The rpm verification step is NOT run, because that is not really compatible with ostree.

Reply:
I moved the "not_installed_docs" to a separate function.

Reply:
Perfect!!